examples/oauth/src/main.rs

```diff
@@ -55,11 +56,11 @@ async fn main() {
         .route("/logout", get(logout))
         .with_state(app_state);
 
-    let listener = tokio::net::TcpListener::bind("127.0.0.1:3000")
-        .await
-        .unwrap();
-    tracing::debug!("listening on {}", listener.local_addr().unwrap());
-    axum::serve(listener, app).await.unwrap();
+    let listener = tokio::net::TcpListener::bind("127.0.0.1:3000").await?;
+    tracing::debug!("listening on {}", listener.local_addr()?);
+    axum::serve(listener, app).await?;
+
+    Ok(())
```

Comment:
I'd still keep the unwraps in main. That way you get a backtrace that points exactly to which line failed. You're also gonna see the errors when you run the app immediately, so returning a result doesn't make much difference.